pkg/apis/componentconfig/types.go

```diff
@@ -397,6 +397,9 @@ type KubeletConfiguration struct {
 	// Currently only cpu and memory are supported. [default=none]
 	// See http://releases.k8s.io/HEAD/docs/user-guide/compute-resources.md for more detail.
 	KubeReserved utilconfig.ConfigurationMap `json:"kubeReserved"`
+	// apiServerList is a list of Kubernetes API servers for publishing events
+	// and reading pods and services. (ip:port), comma separated.
+	APIServerList []string `json:"apiServerList"`
```

Comment:
Ah, wait a minute. I almost forgot :P. One goal of the `KubeletConfiguration` work is to eventually allow us to deprecate Kubelet flags and move to providing local config from a file on disk. So ultimately, anything that is a Kubelet flag should move into the `KubeletConfiguration` structs. So I think it actually makes sense to move the `KubeConfig` field from `KubeletServer` to `KubeletConfiguration`. 

I'm also going to remove `AuthPath`, since that's been marked deprecated since v1.2.